### PR TITLE
Update README.md

### LINKED ISSUE #3214: page not found
**Describe the bug**
[Eventing Kafka Channel Docs](https://github.com/knative-sandbox/eventing-kafka-broker/blob/main/docs/channel/README.md) the links in this file are not found

 **Expected behavior**

 **To Reproduce**

 **Knative release version**

 **Additional context**
Add any other context about the problem here such as proposed priority

### PR DESCRIPTION
**Pull request by akshayamadhuri**
Fixes #3214

### CODE REVIEW

File: docs/channel/README.md [hunk 1]
```diff
@@ -5,9 +5,9 @@
 This repository contains another KafkaChannel implementation, which we believe is more flexible and scalable.
 
 There are already 2 other KafkaChannel implementations:
-the [consolidated KafkaChannel](https://github.com/knative-sandbox/eventing-kafka/blob/main/pkg/channel/consolidated/README.md)
+the [consolidated KafkaChannel]
```
Comment (Leo6Leo) [inline docs/channel/README.md:8]: https://github.com/knative-sandbox/eventing-kafka/tree/v0.36.1/pkg/channel/consolidated

File: docs/channel/README.md [hunk 2]
```diff
@@ -5,9 +5,9 @@
...
 and
-the [distributed KafkaChannel](https://github.com/knative-sandbox/eventing-kafka/blob/main/pkg/channel/distributed/README.md)
+the [distributed KafkaChannel]
```
Comment (Leo6Leo) [inline docs/channel/README.md:10]: https://github.com/knative-sandbox/eventing-kafka/tree/v0.36.1/pkg/channel/distributed

**Review by Leo6Leo — CHANGES_REQUESTED** [review]
Hey @akshayamadhuri, welcome to Knative eventing community, and thanks for your very first contribution! Please refer to my comment above for more information!

**Comment by akshayamadhuri** [comment]
@Leo6Leo thanks for the contribution but i think it needs review

**Comment by Leo6Leo** [comment]
@akshayamadhuri  
https://github.com/knative-sandbox/eventing-kafka-broker/issues/3214#issuecomment-1646115825
https://github.com/knative-sandbox/eventing-kafka-broker/issues/3214#issuecomment-1651888658

File: docs/channel/README.md [hunk 3]
```diff
@@ -21,9 +21,9 @@ The dataplane is designed in a way that it only communicates the control plane v
 ## Comparison with other KafkaChannel implementations
 
 You may find a comparison of this new KafkaChannel implementation with
-the [consolidated KafkaChannel](https://github.com/knative-sandbox/eventing-kafka/blob/main/pkg/channel/consolidated/README.md)
+the [consolidated KafkaChannel]
```
Comment (matzew) [inline docs/channel/README.md:24]: we are not releasing those anymore.

Can we close this PR?

Comment (Leo6Leo) [reply docs/channel/README.md:24]: @matzew Did you mean we are not releasing these feature anymore or not releasing this doc anymore?

If we are not releasing these features anymore, do we need to change this README.md accordingly to reflect this?

If we are not releasing this doc anymore, I will close this PR then

Comment (matzew) [reply docs/channel/README.md:24]: @Leo6Leo ok.
Let's update:
* point to the last release
* say that these _other_ channels are NOT supported 

OR even better: update the doc, and completely remove any mention of those unsupported other channel implementations. Makes sense?

Comment (Leo6Leo) [reply docs/channel/README.md:24]: agreed! How about this we first update the doc to highlight the latest release and clarify that the other channels are not supported. After a transitional period, we'll fully remove mentions of these unsupported channels. I'll create an issue to track this. @matzew

**Comment by Leo6Leo** [comment]
Hey @akshayamadhuri , thanks for reporting this issue and open this PR. As you may see from the discussion above, these channels are not supported by Knative anymore. So would you mind making the following changes?
1. Have a link point to the last release
2. Saying that these other channels are NOT supported and remove their links

**Pull request closed without merging**